styles.css

```diff
@@ -0,0 +1,267 @@
+body {
+    width: 1020px;
+    margin-right: auto;
+    margin-left: auto;
+    font-family: Lato, sans-serif;
+}
+
+/* Header */
+
+.logo {
+    margin-left: 40px;
+}
+
+.header {
+background-color: #2D303A;
+height: 89px;
+display: flex;
+align-items: center;
+justify-content: space-between;
+text-transform: uppercase;
+}
```

Comment:
Could you please fix this section by installing eslint and prettier into VS Code?